bucket/cv4pve-pepper.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.4.0",
+    "description": "Launching SPICE remote-viewer to access VM running on Proxmox VE.",
+    "homepage": "https://github.com/Corsinvest/cv4pve-pepper",
+    "license": "GPL v3",
+    "architecture": {
```

Comment:
There will definitely be dependency to dotnet. Investigate which exactly

Reply:
I have tried the cv4pve-pepper on a freshly installed Windows 10 without dotnet installed, it can run without errors. I checked its build code (https://github.com/Corsinvest/cv4pve-pepper/blob/master/src/Corsinvest.ProxmoxVE.Pepper/build.ps1) it appears it included dotnet when building the exe file. The downloaded exe file is 28MB, it's likely it included dotnet runtime environment.